src/Symfony/Component/Yaml/Inline.php

```diff
@@ -581,6 +584,19 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         }
 
                         return;
+                    case 0 === strpos($scalar, '!php/const:'):
```

Comment:
In that case maybe I should only change the YAML `RuntimeException` to a `ParseException` so it's caught in the DI loader.